app/src/app/routes/Sources/Mappings/MappingHeader.tsx

```diff
@@ -84,6 +84,9 @@ const MappingHeader = (): JSX.Element => {
   const handleDeleteDialogClose = () => {
     setDeleteDialogOpen(false);
   };
+  const handlePreviewDisplay = () => {
+    history.push(`/sources/${sourceId}/mappings/${mappingId}/preview`);
```

Comment:
same here we need to check if `attributeId` is there and format the path accordingly